packages/e2e-tests/plugins/cpt-locking.php

```diff
@@ -52,6 +52,82 @@ function gutenberg_test_cpt_locking() {
 			'template_lock' => false,
 		)
 	);
+	register_post_type(
+		'ul-post-ul-group',
+		array(
+			'public'        => true,
+			'label'         => 'Locked Post Unlocked group',
+			'show_in_rest'  => true,
+			'template'      => array(
+				array(
+					'core/group',
+					array(
+						'templateLock' => false,
+					),
+					array(
+						array( 'core/quote' ),
+						array(
+							'core/paragraph',
+							array(
+								'placeholder' => 'Add a description343',
```

Comment:
Intentional or typo?

```suggestion
								'placeholder' => 'Add a description',
```

Reply:
It was a string I changed to make sure the code executing was the expected one and I missed o revert the change.

Reply:
Hah, it's the same reason for my adding `die( 42 );` in the other PR today. :D